src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);

```

Comment:
move random to the class field, like this example:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/12864128/3f44a98b-5f92-47ef-bc0b-702e3d2b1167)